backend/tests/unit_tests/test_static_util.py

```diff
@@ -0,0 +1,42 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+import pytest
+
+from static.exception import FileNotExistError
+from static.util import (
+    is_image_with_specific_id_exist,
+    delete_image,
+    put_image,
+)
+
+if TYPE_CHECKING:
+    from flask import Flask
+
+_some_image_base64_content = "data:image/png;base64,somecontent"
+_some_image_uuid = "c11d5bcf-f529-4318-904d-4bc8b8d7f68a"
```

Comment:
Use fixture to set up and tear down. Global variables are reused between test cases, which aren't independent.